index.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const serveStatic = require('serve-static');
 const videoStitch = require('video-stitch');
 const fs = require('fs');
+var bodyParser = require('body-parser');
```

Comment:
`const` since we don't want to modify this librairie.

Furthermore, we'll have to remove it when replay filename will be `replay.mp4`, not based on a datetime.